docs/user-accounts-in-8knot.md

```diff
@@ -0,0 +1,38 @@
+# How do user accounts work in 8Knot (OAuth and session handling)
+
+8Knot obviously uses some notion of a user account- there are log in / log out buttons, and user groups are persist across logins.
```

Comment:
"8Knot uses some notion of a user account - there..." or something like that. "Obviously" can come off wrong for new people who this might not be obvious for

Reply:
++